src/tests/Microsoft.PowerFx.Core.Tests/ParseTests.cs

```diff
@@ -862,11 +864,11 @@ public void TestFormulaParseRestart(string script, string key)
             {
                 AllowsSideEffects = false
             };
-            var userDefinitions = UserDefinitions.ProcessUserDefinitions(script, parserOptions, out var userDefinitionResult);
-            Assert.True(userDefinitionResult.HasErrors);
+            var parseResult = UserDefinitions.Parse(script, parserOptions);
+            Assert.True(parseResult.HasErrors);
 
```

Comment:
This is exactly the same as TestFormulasParseError; consider calling that (and changing that method to return the parseResult value